rotkehlchen/exchanges/data_structures.py

```diff
@@ -157,6 +158,13 @@ def serialize(self) -> Dict[str, Any]:
             'notes': self.notes,
         }
 
+    def __str__(self) -> str:
+        return (
+            f'trade at {str(self.location)} location and date '
```

Comment:
nitpick: Capitalize Trade

Reply:
I thought about this but the usual position of the trade I believe is in the middle of a sentence so I decided to use lowercase. In the future I'll capitalize